src/routes/safe_apps/models.rs

```diff
@@ -20,3 +21,27 @@ pub struct SafeAppProvider {
     pub url: String,
     pub name: String,
 }
+
+#[derive(Serialize, Debug, PartialEq, Clone)]
+#[serde(tag = "type")]
+#[cfg_attr(test, derive(serde::Deserialize))]
+pub enum SafeAppAccessControlPolicies {
+    #[serde(rename = "NO_RESTRICTIONS")]
+    NoRestrictions(SafeAppNoRestrictionsPolicy),
+    #[serde(rename = "DOMAIN_ALLOWLIST")]
+    DomainAllowList(SafeAppDomainAllowlistPolicy),
+    #[serde(other)]
+    Unknown,
+}
+
+#[derive(Serialize, Debug, PartialEq, Clone)]
+#[serde(rename_all = "camelCase")]
+#[cfg_attr(test, derive(serde::Deserialize))]
+pub struct SafeAppDomainAllowlistPolicy {
+    pub value: Vec<String>,
+}
+
+#[derive(Serialize, Debug, PartialEq, Clone)]
+#[serde(rename_all = "camelCase")]
+#[cfg_attr(test, derive(serde::Deserialize))]
+pub struct SafeAppNoRestrictionsPolicy {}
```

Comment:
You don't need `{}`, just `replace it with `;` and you should be good :) Then you won't need it in the `converter` method either